src/itemdef.cpp

```diff
@@ -251,6 +254,8 @@ void ItemDefinition::deSerialize(std::istream &is, u16 protocol_version)
 
 		if (readU8(is)) // protocol_version >= 43
 			place_param2 = readU8(is);
+
+		wallmounted_rotate_vertical = readU8(is); // 0 if missing
```

Comment:
Rebase issue. I should've inverted the line `if (readU8(is))` to return early. Now we might read garbage values for protocol_version < 43.